### DISCUSSION TITLE
Have shortened title name as optional variable

### DISCUSSION THREAD

**Discussion by pgomez** [hadynz/obsidian-kindle-plugin#42] [Ideas]
It would be great if there was a way to get the shortened title of a book to use within a note. 

For example:
Atomic Habits: An Easy & Proven Way to Build Good Habits & Break Bad Ones

When synced, the file is named `Atomic Habits`, which is great, but when using the `{{title}}`, it prints out `Atomic Habits: An Easy & Proven Way to Build Good Habits & Break Bad Ones`. If I ever referenced the book in another file, I would prefer to reference the book as `[[Atomic Habits]]` and not the full title.

**Comment by pgomez** [comment]
After I wrote this, I realized I can use `<% tp.file.title %>` from the Templater plug-in to get what I need here.

**Comment by hadynz** [comment]
This is a really good suggestion. I would take it further and suggest that the default book title in the template should be the shortened version.

I've add this as a feature and just released it in the latest version of the plugin v0.2.11.

Thanks for raising @pgomez.
